.mergify.yml

```diff
@@ -17,6 +17,13 @@ queue_rules:
   - name: default
     conditions:
       # Conditions to get out of the queue (= merged)
```

Comment:
These is not the conditions to get out of the queue, but the conditions to get merged.

A pull request gets out of the queue automatically if the pull_request_rules doesn't match or a CI in queue_rules fail/cancelled/error